Calculator7/src/META-INF/MANIFEST.MF

```diff
@@ -1,3 +0,0 @@
-Manifest-Version: 1.0

```

Comment:
Java manifests are specific to a module type. This is not required as soon as the project is run by maven or IDEA.